internal/configuration/configuration.go

```diff
@@ -31,21 +38,39 @@ func generate(cr *v1alpha1.Storage) schema.Configuration {
 		})
 	}
 
+	var keyConfig *schema.KeyConfig
+	if crDb != nil && crDb.Spec.Encryption.Enabled {
+		keyConfig = &schema.KeyConfig{
+			Keys: []schema.Key{
+				{
+					ContainerPath: "/tls/encryption/key",
```

Comment:
TLS abbreviation is irrelevant to disk encryption. Maybe it will be better to put in `/opt/ydb/secrets/database_encryption/key`?